userspace/libsinsp/filterchecks.cpp

```diff
@@ -4461,7 +4464,15 @@ uint8_t* sinsp_filter_check_event::extract(sinsp_evt *evt, OUT uint32_t* len, bo
 						{
 							vector<string> subelements = sinsp_split(e, ':');
 							ASSERT(subelements.size() == 2);
-							m_strstorage = trim(subelements[1]);
+							m_strstorage = subelements[1];
+
+							if (m_strstorage.find("@") != string::npos) {
```

Comment:
Coding style (and the other code in this function) follow the guideline of no space before opening paren and curly braces on their own line. For the sake of consistency, please replicate the style of the existing sinsp code.